.changeset/fuzzy-deers-tease.md

```diff
@@ -0,0 +1,9 @@
+---
+"io-ipatente-practices": patch
+"io-ipatente-licences": patch
```

Comment:
non blocking: for licences i suggest a minor

Reply:
Applied https://github.com/pagopa/io-ipatente/pull/264/commits/08e438216b16544e54f1b1544f546adb5b3731ce